Design-Docs/iam_goals.md

```diff
@@ -7,6 +7,11 @@
   - **Example:** Provide SSO/OAuth2 to access monitoring dashboards and Cortex
   - **Example:** Provide identity aware access to payload K8s clusters (providing K8s `User`/`Group`)
 
+- Provide infrastructure to consumer payloads
+  - **1** prepare Operation Users as Group in Keycloak to managed them selective in scopes 
+  - **2** customize Keystone Polices to prepare trivial Self services for tenants as RAND
```

Comment:
RAND seems to be a n acronym, please spell it out once e.g.: "Random Access Nand Dongle (RAND)"